<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>poseVideoClass</class>
 <widget class="QMainWindow" name="poseVideoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>poseVideo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>961</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>11</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>23</number>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="display_second">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>11</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>23</number>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="progresslabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>600</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current / Total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="startline">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>640</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="endline">
    <property name="geometry">
     <rect>
      <x>744</x>
      <y>640</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="exportLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>640</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current / Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>641</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftexportButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>638</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gotoLine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>640</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoButton">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>639</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightexportButton">
    <property name="geometry">
     <rect>
      <x>915</x>
      <y>638</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>639</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>